lib/Doctrine/DBAL/Platforms/OraclePlatform.php

```diff
@@ -932,9 +932,12 @@ public function usesSequenceEmulatedIdentityColumns()
     public function getIdentitySequenceName($tableName, $columnName)
     {
         $table = new Identifier($tableName);
+        $column = new Identifier($columnName);
 
         // No usage of column name to preserve BC compatibility with <2.5
-        $identitySequenceName = $table->getName() . '_SEQ';
+        // ?????????
+
+        $identitySequenceName = $table->getName() . ($column->getName() !== '' ? '_' . $column->getName() : '') . '_SEQ';
```

Comment:
Unfortunately we cannot do this (see comment about BC above). We included the column name in the sequence naming during the development of 2.5 but decided to revert it to stay backwards compatible. Otherwise existing application will break when upgrading DBAL.